src/sidebar/store/modules/selection.js

```diff
@@ -60,6 +76,46 @@ function initialSelection(settings) {
   return selection;
 }
 
+/**
+ * Configure (user-)focused mode. User-focus mode may be set in one of two
+ * ways:
+ * - A `focus` object containing a valid `user` object is present in the
+ *   application's `settings` object during initialization time
+ * - A `user` object is given to the `changeFocusedUser` action (this
+ *   is implemented via an RPC method call)
+ * For focus mode to be considered configured, it must have a valid `user`.
+ * A successfully-configured focus mode will be set to `active` immediately
+ * and may be toggled via `toggleFocusMode`.
+ *
+ * @param {Object} [focusConfig]
```

Comment:
It would be useful to have a type for whatever `focusConfig` can be. I presume it would be whatever type is used by the `focus` field of the `HostConfig` type defined `types/config.js`? (`HostConfig` is currently missing a definition for the `focus` field)